README.md

```diff
@@ -129,20 +129,24 @@ You can find a further CRUD example in the [example](https://github.com/cloudant
 
 ### Initialization
 
+To use Cloudant, `require('cloudant')` in your code. Then initialize your client in one of the following ways:
```

Comment:
Do you need an 'add', 'use' or some verb? i.e. To use Cloudant, add require('cloudant') to your code